src/sentry/static/sentry/app/views/organizationIntegrations/index.tsx

```diff
@@ -284,50 +286,52 @@ class OrganizationIntegrations extends AsyncComponent<
     );
 
     return (
-      <React.Fragment>
-        {!this.props.hideHeader && <SettingsPageHeader title={t('Integrations')} />}
-        <PermissionAlert access={['org:integrations']} />
-
-        <MigrationWarnings
-          orgId={this.props.params.orgId}
-          providers={this.providers}
-          onInstall={this.onInstall}
-        />
+      <SentryDocumentTitle title={t('Integrations')} objSlug={orgId}>
```

Comment:
Can we move this inside the fragment and not have any `children`? This is more of a stylistic preference so that we remove a layer of nesting.

Reply:
@billyvg already reworking, forgot about this, I'll do it as markstory told just `<SentryDocumentTitle .... />`